src/devices/Media/VideoDevice/Devices/UnixVideoDevice.cs

```diff
@@ -67,10 +66,7 @@ public UnixVideoDevice(VideoConnectionSettings settings)
         /// <param name="path">Picture save path</param>
         public override async Task CaptureAsync(string path)
         {
-            using (CancellationTokenSource cts = new CancellationTokenSource())
-            {
-                await CaptureAsync(path, cts.Token);
-            }
+            await CaptureAsync(path, CancellationToken.None);
```

Comment:
this could be just `return CaptureAsync(path, CancellationToken.None);`

Reply:
(sorry didn't caught it on first pass)

Reply:
> CS1997 C# Since is an async method that returns 'Task', a return keyword must not be followed by an object expression.